brave/src/main/java/brave/propagation/ExtraFieldPropagation.java

```diff
@@ -143,25 +150,24 @@ public FactoryBuilder addPrefixedFields(String prefix, Collection<String> fieldN
     }
 
     public Factory build() {
```

Comment:
How about validating all names in redacted field names are in field names, to catch typos easier.

Reply:
right now adding a redacted field implies adding a field, so that should be covered.